### PR TITLE
Badis cruds done 

### PR DESCRIPTION
**Pull request by Badis1337**

### CODE REVIEW

File: src/services/MessageService.java [hunk 1]
```diff
@@ -8,48 +8,124 @@
...
+        String req = "INSERT INTO `message`( `content`, `seen`, `seen_date`, `date`, `sender_id`, `receiver_id`  )"
+                + " VALUES (?,?,?,?,?,?)";
+
+        pst = CONNECTION.prepareStatement(req);
+
+        pst.setString(1, obj.getContent());
+        pst.setInt(2, (obj.isSeen()) ? 1 : 0);
+        pst.setTimestamp(3, new Timestamp(new Date().getTime()));
+        pst.setTimestamp(4, new Timestamp(new Date().getTime()));
+//        User sender = new User() {
```
Comment (SeifAbdennadher) [inline src/services/MessageService.java:59]: remove comments

File: src/services/MemberService.java [hunk 2]
```diff
@@ -75,13 +84,50 @@ public void update(Member obj) throws SQLException {
...
+        obj.setSmoker(rs.getBoolean("smoker"));
+        obj.setDrinker(rs.getBoolean("drinker"));
+        obj.setMinAge(rs.getInt("min_age"));
+        obj.setMaxAge(rs.getInt("max_age"));
+        obj.setProximity(Enumerations.Proximity.values()[rs.getInt("proximity")]);
+        obj.setLastLogin(rs.getTimestamp("last_login"));
+        obj.setLocked(rs.getShort("locked"));
+        obj.setIp(rs.getString("ip"));
+        obj.setPort(rs.getInt("port"));
+        obj.setAddress(new Address(obj.getId()));
```
Comment (SeifAbdennadher) [inline src/services/MemberService.java:121]: getAddress from addressService
obj.setAddress(AddressService.getInstance().get(new Address(obj.getId()));

File: src/services/MemberService.java [hunk 3]
```diff
@@ -75,13 +84,50 @@ public void update(Member obj) throws SQLException {
...
+
+        String condition = "";
+        if (obj.getId() != 0) {
+            condition = "Where id = " + obj.getId();
+        } else if (obj.getPseudo() != null) {
+            condition = "Where pseudo = " + obj.getPseudo();
+        } else if (obj.getEmail() != null) {
+            condition = "Where email = " + obj.getEmail();
+        }
+        String req = "Select * , date_format( birth_date ,'%d %m %Y') as birth from user " + condition;
```
Comment (elyesmansour) [inline src/services/MemberService.java:96]: now that we're using java.sql.Date we shouldn't encounter problems regarding dates. So you should instead use a regular select query with no date_format function involved. Then to extract the birth_date column, just use rs.getDate("birth_date");

File: src/services/MemberService.java (hunk 2 above)
Comment (elyesmansour) [reply src/services/MemberService.java:121]: you forgot to set the id as well.

Comment (SeifAbdennadher) [reply src/services/MemberService.java:121]: @elyes007 which id? ken u're talking about user_id, donc there is no need for it, since it's already set

**Pull request closed without merging**